homeassistant/components/melcloud/climate.py

```diff
@@ -73,6 +83,18 @@
         True,
     )
 
+    platform = entity_platform.current_platform.get()
+    platform.async_register_entity_service(
+        "set_vane_horizontal",
+        {vol.Required("position"): cv.string},
```

Comment:
Please create a const for the config key e.g. `CONF_POSITION`

Reply:
Done